docs/documentation/quartz-3.x/tutorial/schedule-builder.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Tuning the Scheduler
```

Comment:
should the file name be schedule__r__-builder, `SchedulerBuilder` builds the scheduler but fluent API can build schedules too, can be confusing for sure...